### PR TITLE
Des 319 enable the crear cuenta button when the passwords are the same

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

File: components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css [hunk 1]
```diff
@@ -8,8 +8,7 @@
   height: 3rem;
 }
 
-.registerFormPassword-buttonEnabled,
-.registerFormPassword-buttonSessionDisabled {
+.registerFormPassword-buttonEnabled {
```
Comment (SthefaniaEcheverry) [inline components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css:11]: Change "registerFormPassword-buttonEnabled" to `registerFormPassword-enabledButton`

File: components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css [hunk 2]
```diff
@@ -20,11 +19,6 @@
   color: var(--white);
 }
 
-.registerFormPassword-buttonSessionDisabled:hover {
-  background-color: var(--white);
-  color: var(--turquoise);
-}
-
 .registerFormPassword-buttonDisabled {
```
Comment (SthefaniaEcheverry) [inline components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css:22]: Change "registerFormPassword-buttonDisabled" to registerFormPassword-disabledButton

Comment (Daniielmayo) [reply components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css]: fixed

File: components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css (hunk 1 above)
Comment (Daniielmayo) [reply components/organisms/Register/RegisterFormPassword/RegisterFormPassword.module.css]: fixed

**Review by SthefaniaEcheverry — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]

**Review by DanielCalleCO — APPROVED** [review]

**Pull request merged**